infra/resources/_modules/cosmos/cosmos_etl_messages.tf

```diff
@@ -0,0 +1,25 @@
+module "comsosdb_sql_container_messages_ingestion_test" {
+  source = "github.com/pagopa/terraform-azurerm-v3//cosmosdb_sql_container?ref=v8.28.2"
+
+  name                = "messages-ingestion-test"
+  resource_group_name = var.cosmosdb_account.resource_group_name
+
+  account_name  = var.cosmosdb_account.name
+  database_name = "db"
+
+  partition_key_path = "/id"
+  default_ttl        = "-1"
+}
+
+module "comsosdb_sql_container_messages_ingestion_test_lease" {
+  source = "github.com/pagopa/terraform-azurerm-v3//cosmosdb_sql_container?ref=v8.28.2"
+
+  name                = "messages-ingestion-test-lease"
+  resource_group_name = var.cosmosdb_account.resource_group_name
+
+  account_name  = var.cosmosdb_account.name
+  database_name = "db"
+
+  partition_key_path = "/id"
```

Comment:
Those 2 new container are test containers only. This specific one is the lease test container and i don't really now if it knows fiscalCode as information. Instead the partition key on the same file refers to the messages-test-container that is the one we are going to populate with 100 test messages.

I think this 2 containers must be deleted anyway after the tests are closed because we are going to use the messages container instead and we are going to create a specific lease collection for the ingestion flow. 

@lucacavallaro @Garma00 @acuiuli  make sense ?